src/Raven.Server/Documents/Handlers/Debugging/DebugPackage/DebugPackageAnalyzer.cs

```diff
@@ -0,0 +1,303 @@
+                        IndexesInfo = indexesAnalyzer.IndexesInfo,
+                        TasksInfo = tasksAnalyzer.TasksInfo,
+                    };
+                    
+                    databaseReports.Add(databaseReport);
+                }
+
+                var nodeTag = basicServerInfoAnalyzer.BasicServerInfo?.NodeTag ?? clusterStateAnalyzer.ClusterNodeStateInfo?.Topology?.NodeTag ?? "A";
+                
+                var report = new DebugPackageNodeReport(nodeTag)
+                {
+                    Machine = machineAnalyzer.MachineInfo,
+                    Server = serverAnalysisInfo,
+                    ClusterNode = clusterAnalysisInfo,
+                    Databases = databaseReports.ToArray(),
+                    DetectedIssues = issues,
+                    AnalyzeErrors = errors,
+                };
+
+                nodeReports.Add(report);
+            }
+        }
+
+        var clusterWideIssues = DetectClusterWideIssues(nodeReports);
+        var databaseGroupsIssues = DetectDatabaseGroupsIssues(nodeReports);
+
+        return new DebugPackageReport(nodeReports.ToArray(), new DebugPackageAnalysisIssues
+        {
+            ClusterIssues = clusterWideIssues,
+            DatabaseIssues = databaseGroupsIssues
+        });
+    }
+
+    private List<DetectedIssue> DetectClusterWideIssues(List<DebugPackageNodeReport> reports)
+    {
+        var clusterWideIssues = new List<DetectedIssue>();
+
+        var customElectionTimeoutDetected = false;
+        
+        var clusterLogQueueSize = new Dictionary<string, long>();
+        
+        foreach (var report in reports)
+        {
+            var nodeTag = report.NodeTag;
+            var clusterNodeAnalysis = report.ClusterNode;
+            
+            if (clusterNodeAnalysis != null)
+            {
+                if (clusterNodeAnalysis.ElectionTimeoutInMs != null && clusterNodeAnalysis.DefaultElectionTimeoutInMs != null)
+                {
+                    if (customElectionTimeoutDetected == false)
+                    {
+                        clusterWideIssues.Add(new DetectedIssue("Custom Election Timeout setting defined",
+                            $"The setting '{RavenConfiguration.GetKey(x => x.Cluster.ElectionTimeout)}' is set to " +
+                            $"non default value: {clusterNodeAnalysis.ElectionTimeoutInMs.Value} ms",
+                            IssueSeverity.Warning, IssueCategory.Cluster));
+
+                        customElectionTimeoutDetected = true;
+                    }
+
+                    if (report.Server.NetworkInfo?.PingTestResults != null)
+                    {
+                        foreach (var pingResult in report.Server.NetworkInfo.PingTestResults)
+                        {
+                            if (pingResult.TcpInfo.ReceiveTime > clusterNodeAnalysis.ElectionTimeoutInMs.Value + clusterNodeAnalysis.ElectionTimeoutInMs.Value / 10.0)
```

Comment:
Is this doing what you think it does?
This is raised if the election time is 10% higher than the receive time? Isn't that already super high?

I would say that if the ping time is > 30% of the election timeout - we already have a problem, no?

Reply:
This is raised if the ping time is > 10% of election timeout.